Assets/Scripts/ThrowHead.cs

```diff
@@ -20,7 +20,8 @@ public class ThrowHead : MonoBehaviour {
 	}
 
 	void ThrowRigidBody() {
-		if (head.tag == "HeadIsAttached" && Input.GetButtonDown ("Fire1") ) {
+		float joystickThrow = Input.GetAxis ("JoyPad Fire");
+		if (head.tag == "HeadIsAttached" && (Input.GetButtonDown ("Fire1") || joystickThrow < 0) ) {
```

Comment:
Does this joystickThrow return `-1` if it is being triggered, or should this read `joystickThrow > 0`?